src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -895,6 +900,16 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
             None
         };
 
+        let mut affinity = Affinity {
+            pod_anti_affinity: anti_affinity,
+            pod_affinity,
+            node_affinity,
+            ..Default::default()
+        };
+        if let Some(service_affinity) = &self.config.service_affinity {
+            affinity.merge_from(serde_json::from_str(service_affinity)?);
+        }
```

Comment:
Is there any way we can merge with a provided anti-affinity?

I'm not sure that we'd even want to, but thought it might be worth getting the reason we don't want to written down.

Reply:
i mean, i don't think a blanket anti-affinity configuration really makes any sense? unless you're just trying to use it as an alternative to taints, but then you can just... do that? this seems like another thing that would be better to hold off on until we have an actual use case